src/client/components/Title/TitleDesc.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TitleDesc.styles.css'
+
+const TitleDesc = ({title,description}) =>{
```

Comment:
Fix indentation and styling please. You can set this up in VS Code (if you don't know how reach out to your classmates),